### DISCUSSION TITLE
80s Television (Commercial Broadcast) VHS Setting

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#101] [Presets]
# 80s commercial setting

![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/0f7d039b-d440-48bd-b2cb-379ff5745c9f)

Nothing too damaged, perfect for emulating the 80s commercial look and feel.

## For example, this is a real one

![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/387cc101-1f03-4e6a-ad97-098738995521)
![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/656884f5-09ac-4ccc-bbf7-e98ed2f307ae)

[80s-commercial.json](https://github.com/valadaptive/ntsc-rs/files/15410576/80s-commercial.json)

**Comment by dawnch0rus** [comment]
might be good to bump up the noise - i'd bet that a lot of the noise on those was smoothed out by digital video compression. otherwise this looks pretty nice!

**Reply by Degamisu** [reply]
Thanks! ill make a second version with a TON of improvements. This will be one of them!

**Reply by dawnch0rus** [reply]
if you're interested, i have some lossless screencaps of 80s commercials that ive captured that have a very similar level of degradation. i'm not sure how useful it would be but id love to help out :D

**Reply by Degamisu** [reply]
@nlolnlolnlolnlo That would definitely be useful! i did have some tapes as well that i havent digitized yet. your videos should help.

**Reply by dawnch0rus** [reply]
![vlcsnap-2024-03-26-21h24m23s957](https://github.com/valadaptive/ntsc-rs/assets/154479340/4273ef4b-a328-4293-989d-880cc76a0787)
![vlcsnap-2024-03-26-21h20m52s363](https://github.com/valadaptive/ntsc-rs/assets/154479340/cd456d70-a9c1-4719-9946-65e3ac56ebff)
![vlcsnap-2024-03-26-21h45m41s349](https://github.com/valadaptive/ntsc-rs/assets/154479340/e8f662c4-31e8-4d6a-be18-4eed6a21f841)
![vlcsnap-2024-05-23-17h44m03s731](https://github.com/valadaptive/ntsc-rs/assets/154479340/4669f38d-798d-4512-abee-40cf8565442b)
![vlcsnap-2024-05-23-17h45m26s248](https://github.com/valadaptive/ntsc-rs/assets/154479340/687e1db2-8c67-4316-b48d-874a19f319d7)
![vlcsnap-2024-05-23-17h46m25s199](https://github.com/valadaptive/ntsc-rs/assets/154479340/db24ee93-b5c2-40a4-b204-8418971d5fad)
![vlcsnap-2024-05-23-17h46m47s975](https://github.com/valadaptive/ntsc-rs/assets/154479340/4cccbaed-f6fe-4dbc-85af-664ccd90b5ed)
![vlcsnap-2024-05-23-17h46m50s644](https://github.com/valadaptive/ntsc-rs/assets/154479340/861f6a4c-2caf-4fa2-abab-b71e734b1a63)
![vlcsnap-2024-05-23-17h46m11s218](https://github.com/valadaptive/ntsc-rs/assets/154479340/ffc0403a-27a1-418d-ab1e-fd3252d47311)
![vlcsnap-2024-05-23-17h45m15s298](https://github.com/valadaptive/ntsc-rs/assets/154479340/4d0452d9-5fa8-4995-9ef5-310d07e5e3db)


hope these are helpful :3

**Reply by Degamisu** [reply]
oh yes! these are perfect. What sort of VCR are you using? looks like a hitachi to me.

**Reply by dawnch0rus** [reply]
just my good old mitsubishi hs-u780. it outputs y/c (s-video) with no sort of sharpening or attempted image correction so it's a very honest portrayal of whats on the tape.

**Reply by Degamisu** [reply]
Ah i see. I took some of the pictures and tried to turn them into a setting. it isnt perfect, ill work on it a little more, but this is what it is right now. if you want the setting tell me.

![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/f8349408-c953-4d44-93ab-77097073c055)

**Reply by dawnch0rus** [reply]
it would probably be best to use box filtering for the chroma demodulation since the comb filter has a tendency to smear the lines vertically. maybe even the notch filter since its fairly crunchy

**Reply by Degamisu** [reply]
yup. only problem is that i cant get those horizontal lines. any ideas? i did use interleaving and edge wave.

![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/a302111f-21b2-4afe-be1e-f3f96ac8f024)

**Reply by smfthe1199** [reply]
I want that setting too!

**Reply by Degamisu** [reply]
@valadaptive is there a way that this can be implemented? ive been looking online for the artifact but i cant find it.

**Reply by Degamisu** [reply]
I think i made it work
![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/948b41fd-f74b-4ab8-a4ae-f346747764ec)

**Reply by smfthe1199** [reply]
Thank you!

**Reply by smfthe1199** [reply]
For these!

**Reply by Degamisu** [reply]
Anyone want the current preset?

**Reply by VexorionReal** [reply]
@Degamisu Me, please!

**Reply by smfthe1199** [reply]
@Degamisu Me too, and thanks!

**Reply by Degamisu** [reply]
@VexorionReal @smfthe1199 Sure thing. Here ya go!

(sorry for the late reply, working.)

[hs-u780-2.json](https://github.com/valadaptive/ntsc-rs/files/15443941/hs-u780-2.json)

**Reply by smfthe1199** [reply]
Thank you!

**Comment by smfthe1199** [comment]
Thanks for these!

**Reply by Degamisu** [reply]
No problem. submit what you make!

**Comment by P4ST4N4G4** [comment]
Wow, that look quite realistic, how did you manage to make the colors change so much from the default settings?

**Reply by Degamisu** [reply]
Thank you! I mostly base off of reference and personal experience, and it also helps that I know what each setting does to the image, and why. I try not to go overboard with the effects like some others, which helps out the realism. This software is very feature-packed!
